recipes/PyAV/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```

Comment:
Sorry didn't see your comment when I posted that.

No worries. To be honest, I don't think this should be something that reviewers look for. As noted in that issue, it should either be the linter raising this before merging or the directory name should be ignored in favor of the package name.